skglm/utils/prox_funcs.py

```diff
@@ -30,8 +30,13 @@ def proj_L2ball(u):
 
 
 @njit
-def BST(x, u):
+def BST(x, u, positive=False):
     """Block soft-thresholding of vector x at level u."""
+    if positive:
```

Comment:
I have the same result( by optimality condition in the the prox minimization problem), but do you have a ref or should we include it somewhere in the doc ?

Reply:
I found no ref for the prox of the l2 norm with positive coefficients.
Do you have smth specific in mind? Do you want to recall the formula? or the proofs as well?

I would recall the proxs in the doc, say that the proof can be found [here](https://math.stackexchange.com/questions/1681658/closed-form-solution-of-arg-min-x-left-x-y-right-22-lamb) for the group Lasso with potentially negative coefficients, and can be adapted for positive coefficients.

Reply:
can you include your derivation in a page of the doc, like we did for some other computations ? Like https://contrib.scikit-learn.org/skglm/tutorials/intercept.html#maths-unpenalized-intercept